pom.xml

```diff
@@ -234,9 +234,8 @@
             <inherited>true</inherited>
             <configuration>
               <optimize>true</optimize>
-              <!-- Enable Java 14 for all sources so that Intellij picks the right language level -->
-              <source>14</source>
-              <target>14</target>
+              <!-- Enable Java 14+ for all sources so that Intellij picks the right language level -->
+              <release>${java.vm.specification.version}</release>
```

Comment:
Using this property, in my setup Intellij does not pickup the correct version for language level support
Would putting 14 be acceptable?

Reply:
It works fine for me with maven, but i have not tried with Intellij.

The reason I proposed this change is so that the release version will work with Java versions greater than 14, say e.g. 15 and beyond. `java.vm.specification.version` is a standard Java SE system property, and Maven exposes all properties from java.lang.System, so it is surprising that it is not working in Intellij. It's ok to drop this, for now. We can figure out if/how we want the version to roll later, if needed.